apps/hash-frontend/next.config.js

```diff
@@ -62,6 +62,24 @@ process.env.NEXT_PUBLIC_SENTRY_DSN = process.env.SENTRY_DSN;
 process.env.NEXT_PUBLIC_SENTRY_REPLAY_SESSION_SAMPLE_RATE =
   process.env.SENTRY_REPLAY_SESSION_SAMPLE_RATE;
 
+/**
+ * @todo: import the page `entityTypeId` from `@local/hash-isomorphic-utils/ontology-types`
+ * when the `next.config.js` supports imports from modules
+ */
+const frontendUrl =
+  process.env.NEXT_PUBLIC_FRONTEND_URL ??
+  (process.env.NEXT_PUBLIC_VERCEL_URL
+    ? `https://${process.env.NEXT_PUBLIC_VERCEL_URL}`
+    : process.env.FRONTEND_URL ?? "http://localhost:3000");
+
+const systemUserShortname =
+  process.env.SYSTEM_USER_SHORTNAME ??
+  // you cannot access process.env in NextJS by variable, thus the repetition of keys in this section
+  process.env.NEXT_PUBLIC_SYSTEM_USER_SHORTNAME ??
+  "example-org";
+
+const pageEntityTypeId = `${frontendUrl}/@${systemUserShortname}/types/entity-type/page/v/1`;
```

Comment:
I have suggested elsewhere that if we filter to a specific version we could make that clear in the page title – in that case it's probably worth doing this because we're not actually filtering to a specific version of Page in order to exclude others, there just happens to be only one of them
```suggestion
const pageEntityTypeId = `${frontendUrl}/@${systemUserShortname}/types/entity-type/page/`;
```